resources/views/emails/email_verification.blade.php

```diff
@@ -0,0 +1,16 @@
+@extends('emails.base')
+
+@section('content')
+
+{{ _("Dear")}} {{$user->first_name}},<br>
```

Comment:
This should be one call to `_`. Some languages will start with the name instead of 'Dear'.